### PR TITLE
Added a couple more notes and modifications & modified the tests to accommodate them

### PR DESCRIPTION
**Pull request by AhmedRedaAmin**
In response to get-alex/alex#219

### CODE REVIEW

**Comment by AhmedRedaAmin** [comment]
Thanks again @jenweber for directing me to the node logs , bit of a Travis CI rookie myself unfortunately :disappointed:

File: script/ablist.yml [hunk 1]
```diff
@@ -436,14 +446,19 @@
     - slim
   inconsiderate:
     - anorexic
+    - bony
+    - skinny
```
Comment (wooorm) [inline script/ablist.yml]: Hmm, what was the reason to add skinny here?

Comment (AhmedRedaAmin) [reply script/ablist.yml]: well , good question , the thing is , skinny comes from a description that someone is "skin on bones" which is used to reference people in late stages of starvation , though of course recently it has been normalized in day to day English as a casual word , however, if we are to level this description at someone who actually has anorexia for example , I thought it sounded inconsiderate in that context , or you know , I could just be overthinking this  :'D.
But yeah , here  is how my thought process went for both "bony" and "skinny" .
Anyways , I'll be removing skinny now .

Comment (AhmedRedaAmin) [reply script/ablist.yml]: Also concerning the upper case in `race.yml` , I just capitalized the races ,regions , nationalities and whatnot , I wasn't aware the tooling puts everything to lower case before  matching , so I 'll be on it :+1:

Comment (AhmedRedaAmin) [reply script/ablist.yml]: Finally , it's my pleasure , mate ! :+1:

**Comment by AhmedRedaAmin** [comment]
okay @wooorm , check it out and tell me what you think !

**Comment by AhmedRedaAmin** [comment]
hey @wooorm  , looks like there is a duplicate entry in the main file , it is the one causing my current merge problem 
```
 type: simple
  source: http://ncdj.org/style-guide/
  considerate:
    - person with an intellectual disability
  inconsiderate:
    - intellectually disabled
- type: simple
  source: http://ncdj.org/style-guide/
  note: Assumes that a person with an intellectual disability has a reduced quality of life.
  considerate:
    - person with an intellectual disability
  inconsiderate:
    - suffers from intellectual disabilities
    - suffering from intellectual disabilities
    - suffering from an intellectual disability
    - afflicted with intellectual disabilities
- afflicted with a intellectual disability
```
Guessing from the last merge with @agans

**Comment by wooorm** [comment]
@AhmedRedaAmin Hmm, do you see such a resolve button somewhere in the GH interface? That should be able to resolve your issue!

screenshot 2018-10-08 at 19 34 52

**Comment by AhmedRedaAmin** [comment]
@wooorm , honestly , I thought only you had write privileges to the main repo and so I can't change it personally , however , I just noticed that I can resolve conflicts freely by altering the main repo version (not just my version) and all will be updated in the pull request . (i.e I thought the pull request had only the ability to change my fork and not the main repo )
Like , seriously  , Github keeps surprising me mate :'D . 
I genuinely thought in order to resolve this , I was supposed to pull the main repo again , merge it locally , alter and push (due to the divergence in commit trees)  , but yeah that button was much quicker !

**Comment by AhmedRedaAmin** [comment]
@wooorm could you please merge this mate ?
It is getting really frustrating having to fix merge issues the longer I wait , especially that I have done everything you asked and have no Idea why I am waiting actually , If it isn't needed , I can close the pull request though , but this weird limbo state is really triggering  me , because it's showing on my home feed .

**Comment by wooorm** [comment]
thanks @AhmedRedaAmin! We appreciate it!

**Pull request merged**